workitem/workitemtype.go

```diff
@@ -27,32 +28,38 @@ const (
 	SystemIteration           = "system.iteration"
 	SystemArea                = "system.area"
 
-	// base item type with common fields for planner item types like userstory, experience, bug, feature, etc.
-	SystemPlannerItem = "planneritem"
-
-	SystemUserStory        = "userstory"
-	SystemValueProposition = "valueproposition"
-	SystemFundamental      = "fundamental"
-	SystemExperience       = "experience"
-	SystemFeature          = "feature"
-	SystemScenario         = "scenario"
-	SystemBug              = "bug"
-
 	SystemStateOpen       = "open"
 	SystemStateNew        = "new"
 	SystemStateInProgress = "in progress"
 	SystemStateResolved   = "resolved"
 	SystemStateClosed     = "closed"
 )
 
+// Never ever change these UUIDs!!!
```

Comment:
Right. Tests refer to these UUIDs in order to work just like they used to refer to "system.bug" and "system.feature". What applies to those UUIDs basically also applied to the names we had before. If you would have changed them, you probably created duplicates in the DB or worse.